lib/sinatra/base.rb

```diff
@@ -1691,7 +1691,7 @@ def safe_ignore(ignore)
         unsafe_patterns = unsafe_ignore.map! do |unsafe|
           chars = unsafe.split(//).map! do |char|
             if char =~ /[A-Z]/
```

Comment:
also I think that this and previous `if` unnecessary (if we use `downcase` here). @aidewoode, @kytrinyx what do you think about this?